expansion/src/test/java/no/unit/nva/publication/indexing/ExpandedResourceTest.java

```diff
@@ -724,6 +726,39 @@ void shouldReturnFundingSourceAsObjectWhenFundingSourceLacksLabels() throws Json
         assertFalse(expandedResource.at(JsonPointer.compile("/fundings/0/identifier")).isMissingNode());
     }
 
+    @Test
+    void shouldReturnFundingSourceAsObjectWhenLackingIdentifier() throws JsonProcessingException,
+                                                                         BadRequestException {
+        var publication = randomPublication();
+        var source = URI.create("https://api.test.nva.aws.unit.no/cristin/funding-sources/OTHER");
+        var unconfirmedFunding = (UnconfirmedFunding) new FundingBuilder()
+                                                          .withSource(source)
+                                                          .withFundingAmount(randomMonetaryAmount())
+                                                          .withLabels(randomLabels())
+                                                          // no identifier
+                                                          .build();
+        publication.setFundings(Set.of(unconfirmedFunding));
+
+        var resource = Resource.fromPublication(publication)
+                           .persistNew(resourceService, UserInstance.fromPublication(publication));
+
+        var uriRetriever = FakeUriRetriever.newInstance();
+        FakeUriResponse.setupFakeForType(resource, uriRetriever, resourceService, false);
+
+        var expandedResource = fromPublication(uriRetriever, resourceService, sqsClient, resource)
+                                   .asJsonNode();
+
+        var fundingSource = expandedResource.at(JsonPointer.compile("/fundings/0/source"));
+
+        assertTrue(fundingSource.isContainerNode(),
+                   "Expected funding source to be a object, but got: " + fundingSource);
+        assertEquals("https://api.test.nva.aws.unit.no/cristin/funding-sources/OTHER",
+                     fundingSource.at(JsonPointer.compile("/id")).asText());
+
+        assertFalse(expandedResource.at(JsonPointer.compile("/fundings/0/type")).isMissingNode());
+        assertTrue(expandedResource.at(JsonPointer.compile("/fundings/0/identifier")).isMissingNode());
```

Comment:
This testing is super-involved, can we simply test the JSON to assert positively (rather than the current negative test) that the node equals e.g. 

```
{
    "id": "https://api.test.nva.aws.unit.no/cristin/funding-sources/OTHER"
}
```

If that is our actual expectation

Reply:
I just deleted those lines